inline.go

```diff
@@ -75,6 +76,9 @@ func (p *parser) inline(data []byte) {
 			if p.inlineCallback[data[end]] != nil {
 				if end+1 < len(data) && data[end] == 'h' && data[end+1] != 't' {
 					end++
+				} else if strings.ContainsRune("hHmMfF", rune(data[end])) && !isAutoLink(p, data, end) {
```

Comment:
Agreed. The compiler can inline the function call however, and the cast does not cost anything as far as I know. 

Anyways, I'm working on a generic fix for the useless splits issue which won't require this line :)